tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,204 @@
+# compare across different conditions. There are many measures that can be
+# extracted from ERPs, and many of these are detailed (including the respective
+# strengths and weaknesses) in Ch. 9 of Luck :footcite:`Luck2014` (also see
+# the `Measurement Tool <https://bit.ly/37uydRw>`_ in the ERPLAB Toolbox
+# :footcite:`Lopez-CalderonLuck2014`).
+#
+# This part of the tutorial will demonstrate how to extract three common
+# measures:
+#
+# * Peak Latency
+# * Peak Amplitude
+# * Mean Amplitude
+#
+# Peak Latency and Amplitude
+# --------------------------
+#
+# Probably most common measures of amplitude and latency are peak measures.
+# Peak measures are basically the maximum amplitude of the signal in a
+# specified time window, and the time point (or latency) at which the peak
+# amplitude occurred.
+#
+# Peak measures can be obtained using the `~mne.Evoked.get_peak` method. There
+# are two important things to point out about `~mne.Evoked.get_peak` method.
+# First, it returns the peak latency and amplitude from **all channels** in
+# the `~mne.Evoked` object.
+# Second, the `~mne.Evoked.get_peak` method can find different 'types' of
+# peaks using the ``mode`` argument. There are three options:
+#
+# * ``mode='pos'``: finds the peak with a positive voltage (ignores
+#   negative voltages)
+# * ``mode='neg'``: finds the peak with a negative voltage (ignores
+#   positive voltages)
+# * ``mode='abs'``: finds the peak with the largest absolute voltage
+#   regardless of sign (positive or negative)
 #
+# The following example demonstrates how to find positive peak in the ERP for
+# the left visual condition (i.e., the ``l_vis`` `~mne.Evoked` object). The
+# time window used to search for the peak is between .065 to .115 sec, and all
+# ``'eeg'`` channels are used.
+
+# Get peak amplitude and latency.
+tmin, tmax = .065, .115
+ch, lat, amp = l_vis.get_peak(ch_type='eeg', tmin=tmin, tmax=tmax,
+                              mode='pos', return_amplitude=True)
+
+# Convert latency and amplitude to msec and microvolts
+lat *= 1e3
+amp *= 1e6
+
+# Print output
+print(f'Channel: {ch}')
+print(f'Peak Latency: {lat:.3f} msec')
+print(f'Peak Amplitude: {amp:.3f} \u00B5V')
+
+# %%
+# The output shows that the channel ``eeg55`` had the maximum positive peak in
+# the chosen time window. In practice, one might want to pull out the peak for
+# an *a priori* region of interest or a single electrode depending on the study
+# This can be done by combining the `~mne.Evoked.pick`
+# (or `~mne.Evoked.pick_channels`) methods with the `~mne.Evoked.get_peak`
+# method.
+#
+# Here, let's assume we believe the effects of interest will occur
+# at ``eeg59``.
+
+# Get the peak and latency measure from a single ROI
+# Fist, return a copy of l_vis to select the channel from
+l_vis_roi = l_vis.copy().pick('eeg59')
+_, lat_roi, amp_roi = l_vis_roi.get_peak(tmin=tmin, tmax=tmax, mode='pos',
+                                         return_amplitude=True)
```

Comment:
here I would still store the `ch` return value, and print it in your output message below, just to demonstrate that if you pass in an evoked with only one channel, you get back the correct channel name.  (it's a subtle example of defensive programming, which is good for us to model in our tutorials --- create a situation where a bug would be obvious if it occurred).